File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Grid;
using DevExpress.XtraGrid.Views.Grid.ViewInfo;
using DevExpress.Utils;

namespace WindowsApplication1
{
	public partial class Form1 : Form
	{
				private DataTable CreateTable(int RowCount)
				{
			DataTable tbl = new DataTable();
			tbl.Columns.Add("Name", typeof(string));
			tbl.Columns.Add("ID", typeof(int));
			tbl.Columns.Add("Number", typeof(int));
			tbl.Columns.Add("Date", typeof(DateTime));
			for (int i = 0; i < RowCount; i++)
			{
				tbl.Rows.Add(new object[] {string.Format("Name{0}", i), i, 3 - i, DateTime.Now.AddDays(i)});
			}
			return tbl;
				}


		public Form1()
		{
			InitializeComponent();
			gridControl1.DataSource = CreateTable(20);
			MultiSelectionHelper TempMultiSelectionHelper = new MultiSelectionHelper(gridView1);
		}
	}
}

File: VB/Form1.vb
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Windows.Forms

Namespace WindowsApplication1

    Public Partial Class Form1
        Inherits Form

        Private Function CreateTable(ByVal RowCount As Integer) As DataTable
            Dim tbl As DataTable = New DataTable()
            tbl.Columns.Add("Name", GetType(String))
            tbl.Columns.Add("ID", GetType(Integer))
            tbl.Columns.Add("Number", GetType(Integer))
            tbl.Columns.Add("Date", GetType(Date))
            For i As Integer = 0 To RowCount - 1
                tbl.Rows.Add(New Object() {String.Format("Name{0}", i), i, 3 - i, Date.Now.AddDays(i)})
            Next

            Return tbl
        End Function

        Public Sub New()
            InitializeComponent()
            gridControl1.DataSource = CreateTable(20)
            Dim TempMultiSelectionHelper As MultiSelectionHelper = New MultiSelectionHelper(gridView1)
        End Sub
    End Class
End Namespace
